File: Basics/Basics/PrettyButton.m
//
//  PrettyButton.m
//  Basics
//
//  Created by Jo Albright on 1/5/15.
//  Copyright (c) 2015 Jo Albright. All rights reserved.
//

#import "PrettyButton.h"

@implementation PrettyButton

// Only override drawRect: if you perform custom drawing.
// An empty implementation adversely affects performance during animation.
- (void)drawRect:(CGRect)rect {
    // Drawing code
    
    self.layer.cornerRadius = self.frame.size.height / 2;
    self.layer.masksToBounds = YES;
    
}

@end

File: RandomProjects/AlbumArtApp/AlbumArt/AlbumArt/PrettyButton.swift
//
//  PrettyButton.swift
//  AlbumArt
//
//  Created by Jo Albright on 10/15/15.
//  Copyright © 2015 Jo Albright. All rights reserved.
//

import UIKit

@IBDesignable class PrettyButton: UIButton {

    @IBInspectable var cornerRadius: CGFloat = 0
    
    override func drawRect(rect: CGRect) {

        layer.cornerRadius = cornerRadius
        layer.masksToBounds = true
    
    }

}
